video_worker/config.py

```diff
@@ -24,16 +25,13 @@ def __init__(self, **kwargs):
         self.settings_dict = {}
 
     def run(self):
-        """
-        Read Extant Settings or Generate New Ones
-        """
         if not os.path.exists(self.instance_yaml):
             logger.error('Not Configured')
             return
 
         with open(self.instance_yaml, 'r') as stream:
             try:
                 self.settings_dict = yaml.load(stream)
-            except yaml.YAMLError as exc:
+            except yaml.YAMLError:
```

Comment:
Same as before, we are not utilizing the `exc` alias anywhere so we better also not declare it,

```bash
try:
    config_dict = yaml.load(stream)
except yaml.YAMLError:
    # Log the exception with our message
    logger.exception('default YAML read error')
    return
```